src/game/game_main.cpp

```diff
@@ -85,23 +163,21 @@ void update(f32 delta) {
     }
     Util::end_tweak_section(&show_entity_system);
 
-
-    if (down(Name::UP)) {
-        MyEnt e = {};
-        e.position = random_unit_vec2() * 0.4;
-        e.scale = {0.5, 0.5};
-        auto id = Logic::add_entity(e);
-        LOG("%d %d", id.slot, id.gen);
+    if (pressed(Name::LEFT)) {
+        Renderer::global_camera.position += V2(-1, 0);
```

Comment:
This should probably contain some kind of delta time.

Reply:
Maybe not when checking pressed() instead of down(), but it was mainly used to test adding Vec2s together and then forgotten about.